package.json

```diff
@@ -15,7 +15,7 @@
      "peg": "pegjs --allowed-start-rules Definitions,Type --dependency \"./type-structures\" app/types/type-parser.pegjs && mv app/types/type-parser.js app/types/type-parser-inner.js && sed -i \"\" 's/^var \\.\\/type-structures/var types/' app/types/type-parser-inner.js",
```

Comment:
I realize that this SED script is ridiculous, not sure there's a better way for now. 

This currently copies the '.js' file directly into the "app/" folder. @RighteousRaichu do you know how to fix this?